commands/lastsave.md

```diff
@@ -1,7 +1,7 @@
 Return the UNIX TIME of the last DB save executed with success.
 A client may check if a `BGSAVE` command succeeded reading the `LASTSAVE` value,
 then issuing a `BGSAVE` command and checking at regular intervals every N
-seconds if `LASTSAVE` changed.
+seconds if `LASTSAVE` changed. Redis considers the database saved successfully at startup.
```

Comment:
Let's also mention this default value? Otherwise the sentence looks weird and lacks context
something like your first commit: will return the UNIX TIME that the redis server started.